src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.model.ValidationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final String VALID_LOGIN = "Valid login";
+    private static final String VALID_PASSWORD = "Valid password";
+    private static final String INVALID_BORDERLINE_LOGIN = "user1";
```

Comment:
The INVALID_BORDERLINE_LOGIN constant should be renamed to reflect that it's a login of insufficient length, as per the checklist item about giving constant fields informative names.